airflow/callbacks/callback_requests.py

```diff
@@ -34,8 +34,10 @@ class BaseCallbackRequest(BaseModel):
     :param msg: Additional Message that can be used for logging
     """
 
-    full_filepath: str
+    filepath: str
     """File Path to use to run the callback"""
+    bundle_name: str
+    bundle_version: str | None = None
```

Comment:
hmm, not sure this should have a default value of None.

Reply:
Without the default value, bundle_version is required everytime we want to use the callback

Reply:
Isn't that what we want though? Like it can be None if the bundle doesn't do versioning, but the caller should tell us. Having a default just makes it easier to miss (like it was missed in the triggerer).